pkg/provider/github/github.go

```diff
@@ -432,7 +433,9 @@ func (v *Provider) getPullRequest(ctx context.Context, runevent *info.Event) (*i
 	runevent.BaseBranch = pr.GetBase().GetRef()
 	runevent.HeadURL = pr.GetHead().GetRepo().GetHTMLURL()
 	runevent.BaseURL = pr.GetBase().GetRepo().GetHTMLURL()
-	runevent.EventType = "pull_request"
+	if runevent.EventType == "" {
+		runevent.EventType = triggertype.PullRequest.String()
+	}
```

Comment:
for getPullRequest I believe  evnteType is always `pull_request`

so do we need if validate `""` :thinking: